Pia 2381 add migration guide

a-szotyori opened the pull request:
After PR #2.

a-szotyori committed:
Update Android SDKs

Internal Core API Library 1.0.1 -> 1.4.2
Bank API Library 1.0.1 -> 1.2.0
Capture SDK 1.5.1 -> 1.8.0
Bank SDK 1.4.2 -> 1.7.0

PIA-2272

a-szotyori committed:
Update iOS SDKs

Bank API Library 1.0.0 -> 1.1.1
Bank API Library Pinning 1.0.0 -> 1.1.1
Capture SDK 1.1.0 -> 1.4.0
Capture SDK Pinning 1.1.0 -> 1.4.0
Bank SDK 1.1.0 -> 1.4.0
Bank SDK Pinning 1.1.0 -> 1.4.0

PIA-2272

a-szotyori committed:
Improve readme formatting and grammar

a-szotyori committed:
Add table of contents to readme

a-szotyori committed:
Add section about our new sdks to the readme

a-szotyori committed:
Mention in the readme that PRs are welcomed to extend the iOS proxy

a-szotyori committed:
Add migration section to the readme

PIA-2381

zladzeyka commented on README.md:
```diff
@@ -1,32 +1,122 @@
+  - [Android](#android-2)
+    - [Documentation](#documentation)
+    - [Binding Libraries](#binding-libraries)
+    - [How to use in new project](#how-to-use-in-new-project)
```
"a" article is missing 😉

    - [How to use in a new project](#how-to-use-in-new-project)

zladzeyka commented on README.md:
```diff
@@ -1,32 +1,122 @@
+
+The Gini Bank SDK ([iOS](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/),
+[Android](http://developer.gini.net/gini-mobile-android/bank-sdk/sdk/html/index.html)) builds upon the Gini Capture SDK
+and Gini Bank API Library to enable the photopayment feature while offering additional features like Gini Pay Connect
```
Should we use `photopayment` as one word?

zladzeyka commented on README.md:
```diff
@@ -1,32 +1,122 @@
+
+### Binding Libraries
 
 Because modules depend on each other they need to be build in the right order:
```
... be build -> be built

a-szotyori committed:
Add android build script to create DLLs which are used in the Android example (analog to the iOS example)

a-szotyori committed:
Move iOS build script to root next to the Android one

a-szotyori committed:
Fix Android migration in readme

a-szotyori committed:
Update iOS SDKs

Capture SDK 1.4.0 -> 1.5.0
Capture SDK Pinning 1.4.0 -> 1.5.0
Bank SDK 1.4.0 -> 1.5.0
Bank SDK Pinning 1.4.0 -> 1.5.0

PIA-2272

a-szotyori committed:
Update requirements in readme and add Square.Moshi to Android example app

a-szotyori committed:
Update Android DLLs (built with newer Xamarin.Android version)

a-szotyori committed:
Don't build release configuration of DLLs (weird errors happening)

a-szotyori committed:
Revert to using project references for Android bindings

Using only DLLs didn't work because it would have been necessary for clients to add all the NuGet
packages that our bindings use to their project as well.

Updated the readme as well.

a-szotyori committed:
Fix grammar issues in the readme

zladzeyka reviewed (approved):
🎉👏🏻👍🏻
